### DISCUSSION TITLE
Axios.get is not working well in next13 beta

### DISCUSSION THREAD

**Discussion by berkaycirak** [vercel/next.js#48990] [Help]
### Describe the Bug

In my client components, I am trying to use axios.get method to fetch some data from my api inside app folder but it returns error always. I can use post method and it works well but I couldn't understand why get method is not working. I am able to console log result in the server but on the client side it returns error.
 That is my Client Component
```
'use client';
import React, { useEffect } from 'react';
import axios from 'axios';

const Page = () => {

	useEffect(() => {
		const fetchResult = async () => {
			const results = await axios.get('/api/results');
			console.log(results);
		};
		fetchResult();
	}, []);
```
That is my api below the app folder (app>api>results>route.ts)
```
export async function GET() {
	const currentUser = await getCurrentUser();
	if (!currentUser) {
		return NextResponse.error();
	}

	const results = await prisma.user.findUnique({
		where: {
			email: currentUser.email as string,
		},
	});

	console.log(results);
	NextResponse.json(results);
}
```

**Comment by pierratono** [comment]
@berkaycirak Can you also add the error you are receiving?

**Comment by timneutkens** [comment]
Converted to discussions as the issue template was not followed.

**Comment by ktkennychow** [comment]
Having the same problem, axios doesn't seem to be able to read local apis.

**Reply by berkaycirak** [reply]
Any updates?
